src/web/WebStatusService.cpp

```diff
@@ -188,13 +255,43 @@ void WebStatusService::exportData(AsyncWebServerRequest * request) {
         System::extractSettings(EMSESP_CUSTOMIZATION_FILE, "Customizations", root);
     } else if (type == "entities") {
         System::extractSettings(EMSESP_CUSTOMENTITY_FILE, "Entities", root);
+    } else if (type == "allvalues") {
+        allvalues(root);

```

Comment:
This is not a setting, the `type:allvalues` in the beginning of the json file looks a bit odd. Maybe `root.clear()` before this or in line 241 if (type != "allvalues") {

Reply:
fixed, thanks